aws/lambda/Makefile

```diff
@@ -46,7 +46,7 @@ destroy: configure-state
 build/node/%.zip: node/%/lambda.js node/%/package.json
```

Comment:
I think here we should change the target of this step to be the directory, as it no longer produces the zip. Otherwise this could execute/not execute in the wrong circumstances.

Reply:
I think this build step is still needed to copy a subset of the files to another folder?